.github/workflows/test-pr.yml

```diff
@@ -65,33 +65,6 @@ jobs:
         run: |
           make -C kevm-pyk cov-unit
 
-  kevm-pyk-profile:
-    needs: kevm-pyk-code-quality-checks
-    name: 'Profiling'
-    runs-on: [self-hosted, linux, normal]
-    timeout-minutes: 15
-    steps:
-      - name: 'Check out code'
-        uses: actions/checkout@v3
-        with:
-          submodules: recursive
-          fetch-depth: 0
-      - name: 'Set up Docker'
-        uses: ./.github/actions/with-docker
-        with:
-          container-name: kevm-ci-profile-${{ github.sha }}
-      - name: 'Build kevm-pyk'
-        run: docker exec -u github-user kevm-ci-profile-${{ github.sha }} /bin/bash -c 'make poetry'
-      - name: 'Build targets'
-        run: docker exec -u github-user kevm-ci-profile-${{ github.sha }} /bin/bash -c 'CXX=clang++-14 poetry -C kevm-pyk run kevm-dist --verbose build -j`nproc` plugin foundry'
-      - name: 'Run profiling'
-        run: |
-          docker exec -u github-user kevm-ci-profile-${{ github.sha }} /bin/bash -c 'make profile'
```

Comment:
This also runs `test_gst_to_kore`:

```
$ make profile PROF_ARGS+='--collect-only'
...
collected 2 items

<Package profiling>
  <Module test_foundry_prove.py>
    <Function test_foundy_prove>
  <Module test_gst_to_kore.py>
    <Function test_gst_to_kore>
```

So instead of removing, this should be turned into a job similar to `kevm-pyk-unit-tests`.

Reply:
I've removed `test_foundry_prove.py` from the profiling tests for now.